operation.go

```diff
@@ -11,6 +12,29 @@ import (
 	"github.com/istreamlabs/huma/schema"
 )
 
+// OperationInfo describes an operation. It contains useful information for
+// logging, metrics, auditing, etc.
+type OperationInfo struct {
```

Comment:
Would it be worth including HTTP method here to differentiate operations on the same URITemplate?

Reply:
Good thinking, but you will have the HTTP method from the `*http.Request` object already so you can use that. This was more meant to expose things you can't have access to like the unique Huma-specific operation name and the internal router's URI template that matched.